pages/api/goals/goalsHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from "react";
+import { UserGoalsData } from "../../../graphql/fetchUserGoals";
+
```

Comment:
helpers generally implies that this file will have helper functions. I would rename it types.ts if you need to keep it.

Reply:
I'm also wondering if you need the GoalsSection type at all, since it's identical to the GoalsSectionProps type. I wonder if there's a benefit to having two different types rather than sharing the same one.